config/broadcasting.php

```diff
@@ -36,7 +36,7 @@
             'secret' => env('PUSHER_SECRET'),
             'app_id' => env('PUSHER_APP_ID'),
             'options' => [
-                //
+                'channel' => env('PUSHER_CHANNEL')
```

Comment:
I'm not sure what `channel` is referring to here, or why it's needed. Was this meant to be `cluster`? And if so, can we default to something, like `env('PUSHER_CLUSTER', 'mt1')`?